sdcm/cluster.py

```diff
@@ -1845,6 +1850,24 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):
             self.kill_stress_thread()
             return
 
+        if node.is_ubuntu14():
```

Comment:
More cleaner and efficient way:

```
install_java_script = dedent("""
    apt-get install software-properties-common -y
    add-apt-repository -y ppa:openjdk-r/ppa
    add-apt-repository -y ppa:scylladb/ppa
    apt-get update
    apt-get install -y openjdk-8-jre-headless
    update-java-alternatives -s java-1.8.0-openjdk-amd64
""")
node.remoter.run('sudo bash -cxe "%s"' % install_java_script)
```
`-x`: will print each command that is executed
`-e`: will exit on first failure

Reply:
Fixed.